docs/core-concepts/dnx-migration.md

```diff
@@ -86,19 +86,30 @@ Another way is using the `dotnet` itself to run your code. This is done by provi
 `dotnet path/to/an/assembly.dll`. 
 
 ## Migrating your DNX project to .NET Core CLI
-In addition to using new commands when working with your code, there are two major things left in migrating from DNX:
+In addition to using new commands when working with your code, there are three major things left in migrating from DNX:
 
-1. Migrating the project file (`project.json`) itself to the CLI tooling.
-2. Migrating off of any DNX APIs to their BCL counterparts. 
+1. Migrate the `global.json` file if you have it to be able to use CLI.
+2. Migrating the project file (`project.json`) itself to the CLI tooling.
+3. Migrating off of any DNX APIs to their BCL counterparts. 
+
+### Dealing with global.json file
+The `global.json` file acts like a solution file for both the RC1 and RC2 projects. In order for the CLI tools (as well 
+as Visual Studio) to differentiate between RC1 and RC2, they use the `sdkVersion` to differentiate. If this property is 
```

Comment:
It would be good to say what the sdkVersion is and remove the second differentiate. Like:
The CLI tools use the `sdkVersion` property(?) from the `global.json` file to differentiate between RC1 and RC2.